### PR TITLE
Update SDK support for v0.26.0, facelift for function section

### PR DESCRIPTION
**Pull request by lucaswoj**
screen shot 2016-10-13 at 4 52 34 pm
screen shot 2016-10-13 at 4 52 40 pm

cc @mollymerp for 👁

### CODE REVIEW

File: docs/_generate/index.html [hunk 1]
```diff
@@ -727,32 +727,82 @@ <h3 class='space-bottom1'><a href='#types-function' title='link to function'>Fun
...
-                    <div class=''><var>exponential</var> functions generate an output by interpolating between stops just less than and just greater than the function input. This is the default for <span class='icon smooth-ramp quiet micro space-right indivne' title='continuous'></span> properties.</div>
-                    <div class=''></span><var>interval</var> functions return the output value of the stop just less than the function input. This is the default for <span class='icon step-ramp quiet micro space-right indivne' title='discrete'></span> properties.</div>
-                    <div class=''></span><var>categorical</var> functions return the output value of the stop equal to the function input.</div>
-                </ul>
-              </li>
-            </ul>
+          <div class='col12 pad1x'>
+            <div class="col12 clearfix pad0y pad2x space-bottom2">
+              <div><span class='code'><a id="function-stops" href="#function-stops">stops</a></span></div>
+              <div><em class='quiet'>Required <a href='#types-array'>array</a>.</em></div>
```
Comment (mollymerp) [inline docs/_generate/index.html]: stops are not required if `type === 'identity'`, right? We should remove the _Required_ label I think and explain that caveat here.

**Review by mollymerp — CHANGES_REQUESTED** [review]
Nice work! One small change 😊

File: docs/_generate/index.html (hunk 1 above)
Comment (lucaswoj) [reply docs/_generate/index.html]: Good catch @mollymerp. Fixed now.

**Review by mollymerp — APPROVED** [review]

**Comment by lucaswoj** [comment]
🐑 🐑 🐑 🐑 🐑 🐑 🐑 🐑 🐑 🐑 🐑

**Pull request merged**